### ISSUE TITLE
Unfriendly manual

### ISSUE THREAD

**Issue by ohpyupi** [ui-router/react#32]
I used to do front-end with angular.js. And ui-router was just perfect for me. Because I had a good memory with ui-router in angular.js, I am trying to use ui-router in React too. However, it seems like the manual or documentation is very unfriendly for new people.

Now, I am trying to configure otherwise() function in react, but have no clue what am I supposed to do. Nothing is found in the official document and there are not many resources out there.

**Comment by elboman** [comment]
Hi,
I understand your frustration, documentation is an expensive task e we'd sure appreciate some help :)

Also beware that `ui-router-react` is currently in pre 1.0 and API is changing from time to time to stabilize for version 1.0.

You can find compiled API references on the ui-router website:

https://ui-router.github.io/react/docs/latest/interfaces/url.urlrulesapi.html

We are aiming for a full API reference documentation and a series of tutorials on the website as soon as we hit 1.0.

Let me know if you have any doubt!

**tl;dr:**
```jsx
router.urlService.rules.otherwise('/home');
```

**Comment by ohpyupi** [comment]
Thanks for your helpful answer. I am really looking forward to seeing stablized version and documentation soon!
